rules/manager.go

```diff
@@ -362,6 +362,38 @@ func (g *Group) hash() uint64 {
 	return l.Hash()
 }
 
+// AlertingRules returns the list of the group's alerting rules
```

Comment:
tiny nit: for consistency, godoc comments should be full sentences ending with a `.`

Reply:
Done :)